### PR TITLE
LL-1017 nano X on manager

### PR DESCRIPTION
**Pull request by valpinkman**
Adapt Manager for Nano X

### Type

Feature

### Context

LL-1017

### Parts of the app affected / Test plan

Manager

genuine check
dashboard

### CODE REVIEW

File: static/i18n/en/app.json [hunk 1]
```diff
@@ -283,7 +283,7 @@
     "subtitle": "Install or uninstall apps on your device",
     "device": {
       "title": "Connect your device",
-      "desc": "Follow the steps below to open the Manager",
```
Comment (dasilvarosa) [inline static/i18n/en/app.json]: Keep as is

File: static/i18n/en/app.json [hunk 2]
```diff
@@ -145,10 +145,10 @@
     "messageIfSkipped": "Your {{currencyName}} address has not been confirmed on your Ledger device. Please verify it for optimal security."
   },
   "deviceConnect": {
-    "dashboard": "Navigate to the <1><0>{{managerAppName}}</0></1> on your device",
-    "step1": "Connect and unlock your <1>Ledger device</1>",
```
Comment (dasilvarosa) [inline static/i18n/en/app.json]: Keep as is

File: static/i18n/en/app.json [hunk 3]
```diff
@@ -145,10 +145,10 @@
...
+    "dashboard": "Go to <1>Home screen</1> on your device",
+    "step1": "Connect your <1>Ledger device</1> to your computer and enter your <3>PIN code</3> on your device",
     "step2": "Navigate to the <1><0>{{managerAppName}}</0></1> app on your device",
-    "step3": "Allow <1><0>Ledger Manager</0></1> on your device"
```
Comment (dasilvarosa) [inline static/i18n/en/app.json]: KEep as is

File: src/components/GenuineCheck.js [hunk 4]
```diff
@@ -157,9 +160,9 @@ class GenuineCheck extends PureComponent<Props> {
         id: 'deviceInfo',
         title: (
           <Trans i18nKey="deviceConnect.dashboard" parent="div">
-            {'Navigate to the '}
-            <Bold>{'dashboard'}</Bold>
-            {' on your device'}
```
Comment (dasilvarosa) [inline src/components/GenuineCheck.js]: 1. Keep as is
2. dashboard -> Dashboard

File: src/components/GenuineCheck.js [hunk 5]
```diff
@@ -169,9 +172,9 @@ class GenuineCheck extends PureComponent<Props> {
         id: 'isGenuine',
         title: (
           <Trans i18nKey="deviceConnect.step3" parent="div">
-            {'Allow '}
-            <Bold>{'Ledger Manager'}</Bold>
-            {' on your device'}
```
Comment (dasilvarosa) [inline src/components/GenuineCheck.js]: Keep as is

File: src/components/EnsureDeviceApp.js [hunk 6]
```diff
@@ -98,8 +98,11 @@ class EnsureDeviceApp extends Component<{
             id: 'device',
             title: (
               <Trans i18nKey="deviceConnect.step1" parent="div">
-                {'Connect and unlock your '}
```
Comment (dasilvarosa) [inline src/components/EnsureDeviceApp.js]: Keep as is

File: src/components/GenuineCheck.js (hunk 4 above)
Comment (valpinkman) [reply src/components/GenuineCheck.js]: done

File: src/components/GenuineCheck.js (hunk 5 above)
Comment (valpinkman) [reply src/components/GenuineCheck.js]: done

File: static/i18n/en/app.json (hunk 1 above)
Comment (valpinkman) [reply static/i18n/en/app.json]: done

**Pull request merged**